utils/riff/user_analytic_utils.js

```diff
@@ -324,6 +324,35 @@ class InteractionCounts {
     }
 }
 
+/* ******************************************************************************
+ * getInteractionCountPhrase                                               */ /**
+ *
+ * get either the singular or plural summary phrase for
+ * a given interaction type and count
+ *
+ * @param {string} type - the type of interaction
```

Comment:
Wouldn't it make sense to have the 1st parameter to the getInteractionCountPhrase function actually be an InteractionType, rather than a "random" string. (I know the type here isn't actually random, but since it doesn't match the names in InteractionType it could be anything else that has a locally defined meaning, but in this case why not use a value that has meaning elsewhere.) See the other comment on how that might make things clearer where it is being used.

Which really just means capitalize the property names in the `typePhrases` map so they match the names defined in `InteractionType`.